package-lock.json

```diff
@@ -1,1724 +1,3946 @@
 {
-    "name": "cordova-plugin-media-capture",
```

Comment:
Looks like there were a lot of whitespace changes here. I wouldn't normally care, but this may make merging upstream changes annoying, especially since there shouldn't be any actual dependency changes in our change.

Reply:
I went ahead and `npm install`ed this with Node 14 to keep the diff small. Hopefully Apache updates to Node 16 in the future soon!

(Actually, that could be a good open-source PR since they seem to already be using Node 16 in their CI!)